azure/kubernetes/aks-single-region/helm-values/values-no-domain.yml

```diff
@@ -0,0 +1,104 @@
+---
+identityKeycloak:
+    postgresql:
+        enabled: false
+    externalDatabase:
+        # see https://artifacthub.io/packages/helm/bitnami/keycloak
+        existingSecret: identity-keycloak-secret   # Name of an existing secret resource containing the database credentials
+        existingSecretHostKey: host
+        existingSecretPortKey: port
+        existingSecretUserKey: user
+        existingSecretDatabaseKey: database
+        existingSecretPasswordKey: password
+
+    auth:
+        existingSecret: identity-secret-for-components
+
+global:
+    elasticsearch:
+        enabled: true
+    opensearch:
+        enabled: false
+        aws:
+            enabled: false # we rely on anonymous auth, not irsa, basic auth is not supported on managed OpenSearch
+        auth:
+            username:
+            password:
+        url:
+            protocol: https
+            host: ${OPENSEARCH_HOST}
+            # Amazon OpenSearch Service listens on port 443 opposed to the usual port 9200.
+            port: 443
+
+    identity:
+
+        auth:
+            publicIssuerUrl: http://localhost:18080/auth/realms/camunda-platform  # replace this with a port of your choice when you will do port forwarding
+
+            zeebe:
+                existingSecret:
+                    name: identity-secret-for-components
+            connectors:
+                existingSecret:
+                    name: identity-secret-for-components
+            operate:
+                redirectUrl: http://localhost:8081  # replace this with a port of your choice when you will do port forwarding
+                existingSecret:
+                    name: identity-secret-for-components
+            tasklist:
+                redirectUrl: http://localhost:8082  # replace this with a port of your choice when you will do port forwarding
+                existingSecret:
+                    name: identity-secret-for-components
+            optimize:
+                redirectUrl: http://localhost:8083  # replace this with a port of your choice when you will do port forwarding
+                existingSecret:
+                    name: identity-secret-for-components
+            webModeler:
+                redirectUrl: http://localhost:8084
+            console:
+                redirectUrl: http://localhost:8085
+                existingSecret:
+                    name: identity-secret-for-components
+
+identity:
+    fullURL: http://localhost:8080 # replace this with a port of your choice when you will do port forwarding
+
+    externalDatabase:
+        enabled: true
+        host: ${DB_HOST}
+        port: 5432
+        username: ${DB_IDENTITY_USERNAME}
+        database: ${DB_IDENTITY_NAME}
+        existingSecret: identity-postgres-secret
+        existingSecretPasswordKey: password
+
+webModeler:
+    enabled: true # Enable Web Modeler
```

Comment:
I think the default we have had so far was `false` and you overwrite it in the test